src/pages/Flow/Details-Tile.vue

```diff
@@ -266,16 +269,15 @@ export default {
           <v-list-item dense class="px-0">
             <v-list-item-content>
               <v-list-item-subtitle class="caption">
-                Created by
+                Created <span v-if="hasUser">by</span>
               </v-list-item-subtitle>
-              <div class="subtitle-2">
-                {{
-                  flow.created_by
-                    ? flow.created_by.username
-                    : 'A nonexistent user'
-                }}
+              <div v-if="hasUser" class="subtitle-2">
+                {{ flow.created_by.username }}
+              </div>
+              <div class="caption">
+                <b v-if="!hasUser"> {{ formatTime(flow.created) }} </b>
+                <span v-if="hasUser"> {{ formatTime(flow.created) }} </span>
```

Comment:
I'd alter the class based on the hasUser property.  Something like this: 
```suggestion
             <div class="caption" :class="{ 'font-weight-bold': !hasUser }">
                {{ formatTime(flow.created) }}
              </div>
```